modules/sensor/src/rgb-depth/realsense/vpRealSense_impl.h

```diff
@@ -352,7 +364,7 @@ void vp_rs_get_pointcloud_impl(const rs::device *m_device, const std::map<rs::st
           for (int j = 0; j < depth_width; j++) {
             float scaled_depth = depth[i * depth_width + j] * depth_scale;
 
-            rs::float2 depth_pixel = {(float)j, (float)i};
+            rs::float2 depth_pixel = { (float)j, (float)i };
```

Comment:
Please, introduce static_cast<float> instead of (float)

Reply:
Done in commit 01950712e